packages/edit-site/lib/unbrotli.js

```diff
@@ -180,7 +180,7 @@ module.exports = BrotliBitReader;
      CONTEXT_SIGNED: second-order context model tuned for signed integers.
 
    The context id for the UTF8 context model is calculated as follows. If p1
-   and p2 are the previous two bytes, we calcualte the context as
+   and p2 are the previous two bytes, we calculate the context as
```

Comment:
This file is a copy of an external library, so I don't think any modifications are needed.